src/devices/inovelli.ts

```diff
@@ -2270,6 +2502,51 @@ const definitions: DefinitionWithExtend[] = [
             await reporting.onOff(endpoint2);
         },
     },
+    {
+        zigbeeModel: ['DZM32-SN'],
+        model: 'DZM32-SN',
+        vendor: 'Inovelli',
+        description: 'mmWave Zigbee Dimmer',
+        exposes: exposesListDZM32.concat(identify().exposes as Expose[]),
+        extend: [inovelliExtend.addCustomClusterInovelli(), illuminance(), occupancy()],
+        toZigbee: [
+            tzLocal.light_onoff_brightness_inovelli,
+            tz.power_on_behavior,
+            tz.ignore_transition,
+            tz.identify,
+            tzLocal.inovelli_led_effect,
+            tzLocal.inovelli_individual_led_effect,
+            tzLocal.inovelli_parameters(DZM32_ATTRIBUTES),
+            tzLocal.inovelli_parameters_readOnly(DZM32_ATTRIBUTES),
+        ],
+        fromZigbee: [
+            fz.on_off,
+            fz.brightness,
+            fz.level_config,
+            fz.power_on_behavior,
+            fz.ignore_basic_report,
+            fz.electrical_measurement,
+            fz.metering,
+            fzLocal.inovelli(DZM32_ATTRIBUTES),
+        ],
+        ota: ota.inovelli,
```

Comment:
Can you try: `extend: [light({configureReporting: true}, electricityMeter()]` here? This will allow you to remove the configure, all of fromZigbee/toZigbee except the inovelli ones and light + electricity measurement exposes.